### PR TITLE
SDV2-1359 Enable XDebug on PHP-7.1 image

### PR DESCRIPTION
**Pull request by webit4me**

### CODE REVIEW

File: opg-php-fpm-71-ppa-1604/Dockerfile [hunk 1]
```diff
@@ -27,49 +50,16 @@ RUN apt-get update -y && \
...
-ENV  OPG_PHP_POOL_USER app
-ENV  OPG_PHP_POOL_GROUP app
-ENV  OPG_PHP_POOL_CHILDREN_MAX 4
-ENV  OPG_PHP_POOL_REQUESTS_MAX 0
-ENV  OPG_PHP_POOL_REQUEST_TERMINATE_TIMEOUT 60
-ENV  OPG_PHP_POOL_MEMORY_LIMIT 256M
-ENV  OPG_PHP_POOL_POST_MAX_SIZE 64M
-ENV  OPG_PHP_POOL_UPLOAD_MAX_FILESIZE 64M
-ENV  OPG_PHP_POOL_MAX_INPUT_VARS 1000
+    mkdir -p /data/session && \
```
Comment (carinadigital) [inline opg-php-fpm-71-ppa-1604/Dockerfile]: Any chance of some comments here.

Comment (webit4me) [reply opg-php-fpm-71-ppa-1604/Dockerfile]: Sure, I can try to find out what was its purpose. 
Please note, the last comment is just relocating existing directives and shouldn't add/remove anything

File: opg-php-fpm-71-ppa-1604/Dockerfile [hunk 2]
```diff
@@ -27,49 +50,16 @@ RUN apt-get update -y && \
...
+    phpdismod -v 7.1 xdebug && \
+    echo "xdebug.remote_enable=on\nxdebug.remote_port=9000\nxdebug.remote_host=10.254.254.254" \
+        >> /etc/php/7.1/mods-available/xdebug.ini && \
+    chmod a+x /etc/sv/php-fpm/run && \
+    ln -s /etc/sv/php-fpm /etc/service/ && \
+    echo "<?php echo phpinfo(); ?>" > /app/public/info.php && \
```
Comment (carinadigital) [inline opg-php-fpm-71-ppa-1604/Dockerfile]: Won't this file be exposed publicly?

File: opg-php-fpm-71-ppa-1604/Dockerfile (hunk 1 above)
Comment (webit4me) [reply opg-php-fpm-71-ppa-1604/Dockerfile]: Are we interested on commnet for line 53 ^ only or in general more description about entire RUN's commands?

Comment (carinadigital) [reply opg-php-fpm-71-ppa-1604/Dockerfile]: Generally about the RUN commands. Do what you can.

File: opg-php-fpm-71-ppa-1604/Dockerfile [hunk 3]
```diff
@@ -16,58 +39,34 @@ ARG PHP_PACKAGES="mcrypt \
                   php7.1-pgsql \
                   php7.1-xml"
 
+
 RUN apt-get update -y && \
+# Add ondrej PPA and instal above packages
```
Comment (carinadigital) [inline opg-php-fpm-71-ppa-1604/Dockerfile]: `install` if you fancy correcting a typo

Comment (webit4me) [reply opg-php-fpm-71-ppa-1604/Dockerfile]: thank :)

**Pull request merged**